cuberover:
  config:
    light_pin: 7 # board pin number for the relay control of the light
    pwm_pin: 15

wheel_hwin_settings:
  debug_mode:           False
  send_command_retries: 3
  ros_loop_rate:        50
  right_wheel:          ['joint_right_front_wheel', 'joint_right_back_wheel']
  left_wheel:           ['joint_left_front_wheel', 'joint_left_back_wheel']
  right_addr:           [ 128, 129 ] # note: implement
  left_addr:            [ 128, 129 ] # note: implement
  use_imu0:             True
  use_imu1:             False
  motor_data:
    encoderTicks_per_revolution: 17700.624

roboclaw_settings:
  serial_port:          "/dev/ttyTHS0"
  send_command_retries: 3
  encoder_timeout_ms:   15
  loop_frequency:       50
  baud_rate:            115200
